src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -132,4 +134,16 @@ public void merge_twoRepoConfigs_success() throws ParseException, IOException {
                 actualConfigs.get(0).getAuthorAliasMap().hashCode());
         Assert.assertEquals(REPO_LEVEL_GLOB_LIST, actualConfigs.get(0).getIgnoreGlobList());
     }
+
+    @Test
+    public void test_defaultBranch() throws ParseException, IOException {
```

Comment:
`repoConfig_defaultBranch_success()`.
The `author-config` also needs testing right?

Reply:
This has been tested in the updated system test where the default branch values in author-config.csv has been removed, but a test for author can be added as well inside this method if still required